news/2025/2025-03-25-the-followpoint-special-behind-the-banhammer.md

```diff
@@ -0,0 +1,295 @@
+
+And now with the small introductions done, let's get to the interview!
+
+---
+
+<style>
+    .news-chat-quote__avatar {
+        float: left;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin-left: -50px;
+    }
+
+    .news-chat-quote__text-container {
+        margin-left: 50px;
+    }
+
+    .news-chat-quote__username {
+        font-weight: 600;
+        margin-bottom: 2px;
+    }
+
+    .news-chat-quote__colour-lvd {
+        color: #FFD1DC;
+    }
+
+    .news-chat-quote__colour-gmt {
+        color: #99EB47;
+    }
+</style>
+
+<div class="news-chat-quote__text-container">
+
+![](/wiki/shared/news/2025-03-25-the-followpoint-special-behind-the-banhammer/leaders.png)
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Hello, for starters let's do a question about history! In the past GMT were called Administrators, but why specifically?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>
+
+At one point Admins had [the same usergroup](https://osu.ppy.sh/groups/4) we have now, but that was from before I even heard of the game. Some curious users could refer to [this page](https://web.archive.org/web/20120904074341/https://osu.ppy.sh/wiki/GMT) but this group didn't really have anything to do with the GMT. As for the term itself, "Administrators" must have been used only because of the phpBB default user title for users with the admin control panel.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Why are GMT members' names red in osu!(stable) chat despite the usergroup having green colour?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>
+
+IRC shenanigans. Due to the chat system the stable client is integrated with, all users with an operator status (mod powers) are red, so it's not just us who are red! Admins and devs also have this color and of course the full-red NAT.
```

Comment:
are we going for factual accuracy here or just going with what people think they know about the game's history? because this doesn't sound right to me.

Reply:
Well, that's what Kobold thinks is the reason, but feel free to correct us if that's not why

Reply:
Not sure how to go about this in the text itself tho, will have to ask Kobold I guess